### PR TITLE
Update logging page

### PR DESCRIPTION
**Pull request by kat-walker**
Amended wording in structured logging section, changed Slack channel to #splunk and added contact information.

### CODE REVIEW

File: source/standards/logging.html.md.erb [hunk 1]
```diff
@@ -89,10 +89,9 @@ You should ensure that sensitive information, such as query parameters containin
 In order to allow for rich querying of log data you should ensure that your logs
 are in a structured format.
 
-### Stuctured logging with Splunk
-
-[Splunk] automatically parses JSON log lines. Other formats may need [specific
-field extracts] configured in Splunk.
+[Splunk] can automatically parse many common types of structured data such as CSV, JSON, and XML. 
+A range of add-ons can be found on Splunkbase to parse data from commonly used technologies such as AWS, Azure, Palo Alto Firewalls, and more.
```
Comment (galund) [inline source/standards/logging.html.md.erb:93]: Could we have a link to Splunkbase?

File: source/standards/logging.html.md.erb [hunk 2]
```diff
@@ -125,13 +124,18 @@ There is [broker documentation] describing how drain logs to Splunk via
 The [GOV.UK PaaS Logging] documentation will help you configure Logit and
 drain logs into it from your app.
 
+###Contact
```
Comment (galund) [inline source/standards/logging.html.md.erb:127]:
```suggestion
### Contact
```

File: source/standards/logging.html.md.erb [hunk 3]
```diff
@@ -125,13 +124,18 @@ There is [broker documentation] describing how drain logs to Splunk via
...
+
+Any questions regarding storing and querying logs should be directed to 
+Louise Ryan (`louise.ryan@digital.cabinet-office.gov.uk`) in the first instance.
```
Comment (galund) [inline source/standards/logging.html.md.erb:130]: I feel like this contradicts working in the open - can we not ask @louzoid-gds to monitor #splunk for relevant discussions? :-)

Comment (louzoid-gds) [reply source/standards/logging.html.md.erb:130]: Oh I do already galund :)
Kat - can we drop this Contact part for now?

File: source/standards/logging.html.md.erb (hunk 1 above)
Comment (huwd) [reply source/standards/logging.html.md.erb:93]:
```suggestion
A range of add-ons can be found on [Splunkbase](https://splunkbase.splunk.com/) to parse data from commonly used technologies such as AWS, Azure, Palo Alto Firewalls, and more. 
```

File: source/standards/logging.html.md.erb [hunk 4]
```diff
@@ -1,6 +1,6 @@
 ---
 title: How to store and query logs
-last_reviewed_on: 2022-02-07
+last_reviewed_on: 2024-06-28
```
Comment (erino) [inline source/standards/logging.html.md.erb:3]: This date could be updated to August, or even September - depending on when the PR is merged

**Review by galund — APPROVED** [review]

**Pull request merged**